server/app/views/admin/questions/QuestionsListView.java

```diff
@@ -309,25 +375,41 @@ private Pair<DivTag, ImmutableList<Modal>> renderQuestionCard(
                 "border",
                 ReferenceClasses.ADMIN_QUESTION_TABLE_ROW)
             .with(row)
-            .with(adminNote);
+            .with(adminNote)
+            // Add data attributes used for sorting.
+            .withData(DATA_ATTRIBUTE_ADMIN_NAME, latestDefinition.getName())
+            .withData(
+                DATA_ATTRIBUTE_LAST_MODIFIED,
+                latestDefinition.getLastModifiedTime().orElse(Instant.EPOCH).toString())
+            .withData(
+                DATA_ATTRIBUTE_NUM_PROGRAMS,
+                getTotalNumReferencingPrograms(cardData.referencingPrograms()).toString());
     return Pair.of(rowWithAdminNote, modals.build());
   }
 
+  private Integer getTotalNumReferencingPrograms(
```

Comment:
Optional nit: can this just be a concrete method of `GroupedReferencingPrograms`?

Reply:
Done.